apps/docs/src/examples/extras/stars/App.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+  import { Canvas } from '@threlte/core'
+  import Scene from './Scene.svelte'
+</script>
+
+<div class="relative h-full w-full ">
+  <Canvas>
+    <Scene />
+  </Canvas>
+</div>
```

Comment:
🌟 🌟 Wooh! 🤩 haha nice contribution 😄 

We're trying to avoid tailwind classes in doc examples so that its not required for all folks copy/pasting.

```suggestion
<div>
  <Canvas>
    <Scene />
  </Canvas>
</div>
<style>
  div {
    height: 100%;
  }
</style>
```

Reply:
Ah, fair enough, pretty sure those tailwind classes are also in the other PRs I submitted (Billboard and FakeGlowMaterial) and at least one other example, as I just copy-pasted some other example and changed it slightly.

Will take a look at those in the next few days and open a separate PR for that.

Reply:
great stuff! thank you so much 😃